sklearn/cluster/tests/test_optics.py

```diff
@@ -217,236 +216,168 @@ def test_cluster_sigmin_pruning(reach, n_child, members):
 
 
 def test_reach_dists():
-    # Tests against known extraction array
+    # Expected values, computed with ELKI using:
```

Comment:
Please note down the version you're using. Apart from that, current comment seems enough from my side.